server/server.js

```diff
@@ -252,6 +254,13 @@ app.get('/dashboard/:installationId?', async (req, res) => {
   }
 })
 
+app.get('/buttons-data', async (req, res) => {
```

Comment:
I think this is worth running a few performance tests.

Right now we have about 2500 sessions in production. It would be great to try this out on 100, 1000, 5000, and 10000 sessions.

I'm mostly interested in these things:
* How long does it take?
* Does the time it takes grow linearly or exponentially?
* Are Button presses still being handled while this is running (assuming that it does run for long enough to try this out)

Reply:
This has been done, more info/conversation in this slack thread: https://bravely.slack.com/archives/C01KLE54K7Y/p1614972901043600